Add reporting-guide (#90)
pythonindia/inpycon2024: src/reporting-guide.md

---
layout: content.liquid
title: Reporting Guide
---

### Reporting Process

If you experience any harassment, or unfair treatment, or witness any behavior you believe violates the PyCon India [Code Of Conduct](https://in.pycon.org/2024/code-of-conduct/), we encourage you to report it as soon as possible. Reports can be made through the following channels:

- **Code of Conduct (CoC) Workgroup Area:** You can directly approach the CoC workgroup area with any concerns you may have. If you are unable to locate the CoC workgroup area, please ask one of our volunteers wearing Volunteer badges for assistance. They will gladly direct you to the CoC Workgroup area.

- **Email:** You can send an email to [coc@in.pycon.org](mailto:coc@in.pycon.org) to submit your report electronically.

> - **In-person:** If you prefer to discuss the matter in-person, please notify any member of the PyCon India organizing team or the Code Of Conduct Committee.

Reviewer: I think to write this line, we need to mention somewhere in this page who are the ones in the CoC Committee?

Author: Yeah adding the COC Committee members' contact details.

---

> slug: reporting-guide

Reviewer 2: add `layout: content.liquid`

Author: Added.

title: Reporting Guide
---

> At PyCon India 2024, we are committed to providing a safe and welcoming environment for all attendees. We take incidents of harassment, misconduct, or violations of our [Code Of Conduct](../code-of-conduct) very seriously. This Reporting Guide aims to provide clear instructions on how to report any concerns and the steps we will take to address reported incidents.

Reviewer 3: ```suggestion
[Code Of Conduct](https://in.pycon.org/2024/code-of-conduct/) 
```

### Reporting Process

> If you experience any harassment, or unfair treatment, or witness any behavior you believe violates the PyCon India [Code Of Conduct](../code-of-conduct), we encourage you to report it as soon as possible. Reports can be made through the following channels:

Reviewer 3: use absolute URL as mentioned earlier.

> a. **Code of Conduct (CoC) Workgroup Area:** You can directly approach the CoC workgroup area with any concerns you may have. If you are unable to locate the CoC workgroup area, please ask one of our volunteers wearing "PyCon India Volunteer" badges for assistance. They will gladly direct you to the CoC Workgroup area.

Reviewer 3: s/PyCon India Volunteer/Volunteer/

- **Date, time, and location of the incident:** Be as specific as possible about when and where the incident occurred.

- **Description of the incident:** Provide a detailed account of what happened, including any offensive language or actions involved.

- **Evidence, if available:** If there is any publicly available evidence (e.g., screenshots, recordings, links to relevant discussions), please include them in your report.

- **Additional context:** Share any context you believe might be relevant to the incident.

- **Is the incident ongoing?:** Let us know if the incident is ongoing or if there is an immediate threat to someone's safety.

### Confidentiality

All reports will be treated with strict confidentiality. We understand the sensitive nature of such matters and will protect the privacy of all parties involved to the best of our ability. In some cases, we may need to make a public statement, but the identities of victims and reporters will remain confidential unless instructed otherwise.

### Reporting Concerns about Volunteers or Organizers

> If your report concerns a possible violation by a volunteer or organizer, you can make your report directly to the Chair of the Code Of Conduct Committee, whose contact information will be made available during the conference.The current Chair is Nivesh Krishna, and you can reach out via email at vniveshkrishna.98 at gmail dot com

Reviewer 3: ```suggestion
If your report concerns a possible violation by a volunteer or organizer, you can make your report directly to the Chair of the Code Of Conduct Committee, whose contact information will be made available during the conference.The current Chair is Sayan Chowdhury, and you can reach out via email at 911 AT yudocaa DOT in
```

### Response and Resolution

> Once a report is received, the Code Of Conduct Committee will promptly initiate an investigation. The Diversity working group will review the incident, consider all the information provided, and determine the appropriate response. Possible responses may include but are not limited to:

Reviewer 3: ```suggestion
Once a report is received, the Code Of Conduct Committee will promptly initiate an investigation. The Code of Conduct Working Group will review the incident, consider all the information provided, and determine the appropriate response. Possible responses may include but are not limited to:
```

> - No violation found: If the incident does not violate the [Code Of Conduct](../code-of-conduct).

Reviewer 3: absolute URL
